docs/resources/policy_rule_list.md

```diff
@@ -35,9 +35,8 @@ description: |-
 Required:
 
 - **priority** (Number)
+- **rule_id** (Number)
```

Comment:
Please modify the source file and fix the field properly from `required` to `optional` and re-generate the this doc